packages/component-library/styles/color.scss

```diff
@@ -1,63 +0,0 @@
-@import "~@kaizen/design-tokens/sass/color-vars";
```

Comment:
This is so good to see! 👏🏻

Reply:
As excited as this makes me, I do wonder if it's _potentially_ a little heavy-handed. Could we release the Stylelint rule first before making this breaking change?

Reply:
That's a really good point. My thinking was just that we couldn't release the breaking change without a Stylelint rule, but there's no reason we couldn't release the stylelint plugin first. I'd like us to get to the point where we're so confident with automatic breaking change updates that it's not even an issue, but yeah, probably best to err on the safer side at this point!

Reply:
The other upside to just releasing the stylelint rule is that even if consumers don't want to upgrade right way, consumers will stop getting thousands of annoying scss warnings.

Reply:
I have split out the actual breaking change into a separate  PR (https://github.com/cultureamp/kaizen-design-system/pull/1950), which I will rebase and fix up once this is merged.